cli/cmd/install.go

```diff
@@ -304,54 +172,50 @@ control plane. It should be run after "linkerd install config".`,
   # Install Linkerd into a non-default namespace.
   linkerd install control-plane -l linkerdtest | kubectl apply -f -`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !options.skipChecks {
+			if !skipChecks {
 				// check if global resources exist to determine if the `install config`
 				// stage succeeded
-				if err := errAfterRunningChecks(options); err == nil {
+				if err := errAfterRunningChecks(values.Global.CNIEnabled); err == nil {
 					if healthcheck.IsCategoryError(err, healthcheck.KubernetesAPIChecks) {
-						fmt.Fprintf(os.Stderr, errMsgCannotInitializeClient, err)
-					} else {
 						fmt.Fprintf(os.Stderr, errMsgGlobalResourcesMissing, controlPlaneNamespace)
 					}
 					os.Exit(1)
 				}
 			}
-
-			if !options.ignoreCluster {
+			if !ignoreCluster {
+				// Ensure there is not already an existing Linkerd installation.
 				if err := errIfLinkerdConfigConfigMapExists(cmd.Context()); err != nil {
 					fmt.Fprintf(os.Stderr, errMsgLinkerdConfigResourceConflict, controlPlaneNamespace, err.Error())
 					os.Exit(1)
 				}
 
 			}
 
-			return installRunE(cmd.Context(), options, controlPlaneStage, flags)
+			return install(cmd.Context(), os.Stdout, values, flags, controlPlaneStage)
 		},
 	}
 
-	cmd.PersistentFlags().BoolVar(
-		&options.skipChecks, "skip-checks", options.skipChecks,
-		`Skip checks for namespace existence`,
-	)
```

Comment:
This wasn't add back after the refactoring, even though there's a referrence to a `skipChecks` var